packages/angular/src/generators/library/lib/add-standalone-component.ts

```diff
@@ -8,38 +8,41 @@ import { addChildren } from './add-children';
 
 export async function addStandaloneComponent(
   tree: Tree,
-  options: NormalizedSchema
+  { libraryOptions, componentOptions }: NormalizedSchema
 ) {
   await componentGenerator(tree, {
-    name: options.name,
+    ...componentOptions,
+    name: componentOptions.name,
     standalone: true,
     export: true,
-    project: options.name,
+    project: componentOptions.name,
```

Comment:
Should this be `libraryOptions.name` instead?

Edit: I saw it has the same value as the `componentOptions.name` but it feels more correct to use the library one since we are assigning the project name.

Reply:
I wanted to have it so that any code interacting with the component was coming from the componentOptions, to keep it somewhat clear? But I can change. What do you think?

Reply:
Yeah, I understood the intention of separating those, but when we refer to `project` we are referring to the library name, so it'd be better to understand if `libraryOptions.name` is used here. In the future, for whatever reason, if the component name is decided to be something different than the library name, this code would be wrong.

Reply:
Oh I see what you mean! Good point